MMM-Bring.js

```diff
@@ -19,43 +20,90 @@ Module.register("MMM-Bring", {
     },
 
     start: function () {
-        this.list = undefined;
         this.sendSocketNotification("GET_LIST", this.config);
         setInterval(() => {
             this.sendSocketNotification("GET_LIST", this.config);
         }, this.config.updateInterval * 60 * 1000);
+        this.lists = ["Liste1", "Liste2"];
+    },
+
+    createDropDown: function() {
+        const drop = document.createElement("div");
+        drop.className = "bring-dropdown-title";
+        const titleBtn = document.createElement("input");
+        titleBtn.setAttribute("type", "button");
+        titleBtn.className = "bring-titleBtn bright";
+        titleBtn.value = this.config.listName +  " \u2BC6";
+        titleBtn.addEventListener("click", function () {
+            document.getElementById("bring-dropItems").classList.toggle("show");
+        });
+        const dropList = document.createElement("div");
+        dropList.id = "bring-dropList";
+        const dropItems = document.createElement("div");
+        dropItems.id = "bring-dropItems";
+        var self = this;
+        for (var i = 0; i < this.lists.length; i++) {
+            var dropItem = document.createElement("div");
+            dropItem.className = "bring-dropItem";
+            dropItem.innerHTML = this.lists[i].name;
+            dropItem.addEventListener("click", function() {
+                self.config.listName = this.innerHTML;
+                self.sendSocketNotification("GET_LIST", self.config);
+            });
+            dropItems.appendChild(dropItem);
+        }
+        dropList.appendChild(dropItems);
+        drop.appendChild(titleBtn);
+        drop.appendChild(dropList);
+        return drop;
     },
 
     getDom: function () {
         const container = document.createElement("div");
         container.className = "bring-list-container bring-" + this.data.position;
 
-        if (this.config.showListName && this.list && this.list.name) {
-            const title = document.createElement("h3");
-            title.innerText = this.list.name;
-            container.appendChild(title);
+        if (this.config.showListName && this.currentList && this.currentList.name) {
+
+            const dropTitle = this.createDropDown();
+            document.addEventListener("click", event => {
+                if (!event.target.matches('.bring-titleBtn')) {
+                    var dropDown = document.getElementById("bring-dropItems");
+                    if (dropDown.classList.contains('show')) {
+                        dropDown.classList.remove('show');
+                    }
+                    /*var dropdowns = document.getElementsByClassName("dropdown-content");
```

Comment:
can be removed?

Reply:
Oh sorry I accidentally resolved this. Will have a look into it.

Reply:
Removed.